index.html

```diff
@@ -0,0 +1,17 @@
+//file <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="styles/style.css">
+
+    </head>
+    <title></title>
+  </head>
+  <body>
+    <link rel="stylesheet" type="text/css" href="scripts/script.css">
+
+    <h1> Hi
+    </h1>
+    <img src="puppy.gif" alt="puppy"height="100" width="100">
```

Comment:
One more comment... What your code is saying here is, look for a file called puppy.gif in the same folder where my index.html file is. Since you don't have that file in the folder, it doesn't know what to display here.

Instead, try making your `src` equal to an image on the internet - you will need to reference the entire URL for the computer to be able to find it. My favorite puppy picture is this one: `https://www.theurbanlist.com/content/article/dogs-to-follow-on-instagram.png`